content/docs/identity-providers/cognito.mdx

```diff
@@ -75,7 +75,8 @@ If you need to make changes after creating your pool, be aware that some setting
    | **Field** | **Description** |
    | --- | --- |
    | Enabled Identity Providers | Choose **Cognito User Pool**, unless you have set up another **Identity Provider** (eg SAML) |
-   | Callback URL(s) | https://authenticate_service_url/oauth2/callback |
+   | Allowed sign-out URLs | `https://{AUTHENTICATE_DOMAIN}/.pomerium/signed_out` |
+   | Callback URL(s) | `https://authenticate_service_url/oauth2/callback` |
```

Comment:
We should pick a single placeholder for the authenticate domain here, right? Let's pick just one of either `{AUTHENTICATE_DOMAIN}` or `authenticate_service_url` and use it consistently.

```suggestion
   | Allowed callback URLs | `https://{AUTHENTICATE_DOMAIN}/oauth2/callback` |
   | Allowed sign-out URLs | `https://{AUTHENTICATE_DOMAIN}/.pomerium/signed_out` |
```

It might even make sense to emphasize that users need to replace this placeholder with the actual domain they're using, e.g. on line 73:

> In the settings for **Pomerium** app, put in the following details (replace `{AUTHENTICATE_DOMAIN}` with the actual domain you will use for the [Authenticate Service URL](https://www.pomerium.com/docs/reference/authenticate-service-url)):

Reply:
@kenjenkins good call, I implemented it.